app/models/response.rb

```diff
@@ -2,19 +2,20 @@
 #
 # Table name: responses
 #
-#  id             :bigint           not null, primary key
-#  comment        :text
-#  gif_url        :string
-#  not_working    :boolean          default(FALSE)
-#  notices        :jsonb
-#  productivity   :integer
-#  rating         :integer
-#  steps          :string           not null
-#  created_at     :datetime         not null
-#  updated_at     :datetime         not null
-#  emotion_id     :bigint
-#  time_period_id :bigint           not null
-#  user_id        :bigint           not null
+#  id                 :bigint           not null, primary key
+#  bad_follow_comment :text
+#  comment            :text
+#  gif_url            :string
+#  not_working        :boolean          default(FALSE)
+#  notices            :jsonb
+#  productivity       :integer
+#  rating             :integer
+#  steps              :string
```

Comment:
Must be with not null

Reply:
@VitaliiShevchenko This is an auto-generated comment dependent on migrations that are not related to this PR

Reply:
If you can please fix it

Reply:
It is not good idea change old migration

Reply:
Now we have correctly of the step field in the master. Your auto-generated migration breaked this field. Please think good who must fix it

Reply:
@VitaliiShevchenko Apparently, this data in the master was entered manually through adding into `schema.rb`:
`t.string "steps", null: false`
 this was done incorrectly. 
 I am against making manual changes to automatically generated code.
 In order to make field "steps"  `null: false` you should make a new migration.

Reply:
@rogergraves please resolve our differences

Reply:
Since @SerhiiBorozenets has a valid point of view about changing generated code he is not obligated to change it. However if @VitaliiShevchenko wishes he can push a correction in a new PR.

Reply:
For what it's worth, we do not use this model annotation in the main Cloverpop app. I expect you will find it less helpful as time goes on and you have more familiarity with the schema and gain more experience.